guillotina/contrib/catalog/pg.py

```diff
@@ -487,6 +488,7 @@ def get_default_where_clauses(self, container: IContainer) -> typing.List[str]:
         users.extend(principal.groups)
         roles_dict = policy.global_principal_roles(principal.id, principal.groups)
         roles.extend([key for key, value in roles_dict.items() if value])
+        roles = get_roles_principal(container)
```

Comment:
You are overriding the `roles` value used above. Is that on purpose?

Reply:
Should it be an extension?

Reply:
ops I forgot to delete it, get_roles_principal already gets the global ones, so its not an extend

Reply:
ahh right, that makes sense.

Reply:
Fixed at https://github.com/plone/guillotina/commit/bd1468066189c8fe95b52a1f0002070b54aaeafd (sorry I did not do a PR)